target/linux/octeon/base-files/etc/board.d/01_network

```diff
@@ -7,13 +7,15 @@
 
 board_config_update
 
-case "$(board_name)" in
-itus,shield-router)
-	ucidef_set_interfaces_lan_wan "eth1 eth2" "eth0"
-	;;
-*)
-	ucidef_set_interfaces_lan_wan "eth0" "eth1"
-	;;
+board="$(board_name)"
+
+case "${board}" in
+	itus,shield-router)
+		ucidef_set_interfaces_lan_wan "eth1 eth2" "eth0"
+		;;
+	*)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
+		;;
```

Comment:
I'm not a fan of this kind of cosmetic changes, at least if they happen isolated like here. There is no clear better or worse here and effectively, somebody could just change it back for the same "reason" in two years ...

Please remove this kind of changes from the commit.

Reply:
by 'no cosmetic changes' you mean:
- no curly brackets
- do not add tab ident where it was not there before (although openwrt use tabs ident everywhere?)

and that seems to be it?